skops/io/_cli.py

```diff
@@ -0,0 +1,113 @@
+from __future__ import annotations
```

Comment:
I think it'd make sense to move everything related to CLI to a `skops._cli` module and have them all in the same place. WDYT?

Reply:
I agree with that, that makes sense to me. I'll refactor to add this